### DISCUSSION TITLE
ModuleNotFoundError: No module named 'facexlib'

### DISCUSSION THREAD

**Discussion by tangleps112** [lllyasviel/Fooocus#915] [Q&A]
After completing the automatic upgrade, the command window will display this message and it will no longer work. How can I solve this problem?ModuleNotFoundError: No module named 'facexlib'

**Comment by lllyasviel** [comment]
show full log

**Reply by tangleps112** [reply]
I have downloaded the compressed package again and it has been overwritten. It is estimated that there was a conflict in the previous version

**Reply by tangleps112** [reply]
Thank you for your response

**Reply by pyrowood** [reply]
I have the same issue.

**Reply by tangleps112** [reply]
downloaded the compressed package again

**Reply by tangleps112** [reply]
I have already resolved it
